README.md

```diff
@@ -126,6 +126,49 @@ require("arrow.persist").load_cache_file()
 
 Obs: persistence.nvim works fine with arrow.
 
+## Commands Module
```

Comment:
i think this commands explanation don't belongs to the README (can confuse the new comers)

Could be on the native help system on neovim

Reply:
I'll create a new pr with contributing guide with it 👀